lib/active_job/plugins/resque/solo/inspector.rb

```diff
@@ -21,14 +23,18 @@ def self.resque_present?
             ActiveJob::Base.queue_adapter.is_a? ActiveJob::QueueAdapters::ResqueAdapter
           end
 
+          def self.resque_scheduler_present?
+            defined? ::Resque::Scheduler
+          end
+
           def around_enqueue(job, block)
-            if Inspector::resque_present?
+            if Inspector.resque_present?
 
-              Lock.try_acquire_release(lock_key(job)) do |lock, extend_at|
+              Lock.try_acquire_release(lock_key('enqueued', job)) do |lock, extend_at|
```

Comment:
Why is the extra parameter needed for `lock_key()`?  Is it being using used elsewhere with a different argument than 'enqueued'?